creator-node/src/services/sync/syncUtil.ts

```diff
@@ -138,5 +177,33 @@ async function fetchExportFromNode({
   }
 }
 
-export { fetchExportFromNode }
-export type { FetchExportParams, FetchExportOutput }
+async function getSyncStatus(syncUuid: string) {
+  return redisClient.get(`syncStatus${syncUuid}`) as Promise<SyncStatus | null>
+}
+
```

Comment:
question for ya - should these uuids be cleared on service restart?
in theory other node could still be polling for this sync but also this node won't process that sync any more so why keep that redis key around
smth to think about

Reply:
true, I just relied on expiry but didn't think of that edge case of the node restarting while the other node still polls. easy enough change - added that now

Reply:
dope